docs/readthedocs/source/doc/PPML/Overview/install.md

```diff
@@ -0,0 +1,54 @@
+# PPML Installation
+
+---
+
+#### OS requirement
+
+
+```eval_rst
+.. note::
+    **Hardware requirements**:
+
+     Intel SGX: PPML's features (except Homomorphic Encryption) are mainly built upon Intel SGX. Intel SGX requires Intel CPU with SGX feature, e.g., IceLake (3rd Xeon Platform). [Check if your CPU has SGX feature](https://www.intel.com/content/www/us/en/support/articles/000028173/processors.html).
```

Comment:
As here the contents for note box is in `eval_rst` block, we need to use rst syntax for all the contents inside it. Thus, for the hyperlink, we should use
```
`Check if your CPU has SGX feature <https://www.intel.com/content/www/us/en/support/articles/000028173/processors.html>`_
```